### ISSUE TITLE
[Accessibility] Slides Page: Select element doesn't have an accessible name

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#1019]
**Describe the bug**
The "select" element does not have an accessible name.

**To Reproduce**
1. Go to 'Slides'
2. Right-click on page and click "inspect"
3. Run a a11y test using the Axe devtools.

**Expected behavior**
The "select" element must have an accessible name.

**Screenshots**
![image](https://github.com/user-attachments/assets/f247a509-0f64-479d-b141-33f046ab2935)

**Comment by nwanduka** [comment]
Hi @YashrajKupekar17, thanks for volunteering to fix this. There's already a PR that fixes this issue. You could check out other issues that haven't been fixed yet.
